projects/docs/docs/guides/third-party-dependencies.md

```diff
@@ -18,9 +23,10 @@ Learn how to define the contract between the dependency managers (Cocoapods, Car
 ### Step 1: "Dependencies.swift"
 
 1. Create `Dependencies.swift` in your project in `Tuist/` directory.
-2. Run `tuist edit` to start editing your manifest files.
-3. Add `import ProjectDescription` on the top of file.
-4. Define your dependencies.
+2. You can read more about `Dependencies.swift` [here](/manifests/dependencies/).
+3. Run `tuist edit` to start editing your manifest files.
+4. Add `import ProjectDescription` on the top of file.
+5. Define your dependencies.
 
 Example of project structure with added `Depedencies.swift` manifest file:
```

Comment:
Should we add some thoughts on what should be `.gitignore`ed? I think there are some files we should tell people to commit (like the lock files, etc.). But the build artifacts shouldn't be committed right?

Reply:
"Committing or no committing build artifacts" is still unresolved issue. The developers with different experience are still discussing it. Some of arguments can be read [here](https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control).

I don't want to suggest an approach in documentation and leave the decision to the users.

Reply:
I agree that we should not _advise_ people to gitignore or not. That being said, I'd include here a note saying something like "If you don't want to check in the downloaded dependencies, put the following in your `.gitignore`". As of right now, it might not be immediately obvious what to `.gitignore`, so we should help users if they are in the "club" of gitignoring dependencies (which I am a member of 🤠)